Sources/Logging/Logging.swift

```diff
@@ -505,11 +505,71 @@ public struct MultiplexLogHandler: LogHandler {
     }
 }
 
-/// Ships with the logging module, really boring just prints something using the `print` function
-internal struct StdoutLogHandler: LogHandler {
+/// A wrapper to facilitate `print`-ing to stderr and stdio that
+/// ensures access to the underlying `FILE` is locked to prevent
+/// cross-thread interleaving of output.
+private struct FileOutputStream: TextOutputStream {
+    let file: UnsafeMutablePointer<FILE>
+
+    func write(_ string: String) {
+        string.withCString { ptr in
+            flockfile(file)
+            defer {
+                funlockfile(file)
+            }
+            _ = fputs(ptr, file)
+        }
+    }
+}
+
+// Prevent name clashes
+#if os(macOS) || os(tvOS) || os(iOS) || os(watchOS)
+let systemStderr = Darwin.stderr
+let systemStdout = Darwin.stdout
+#else
+let systemStderr = Glibc.stderr!
+let systemStdout = Glibc.stdout!
```

Comment:
@ktoso You recommended the local definitions of the system `stderr` but I found that `Glibc.stderr` is an IOU. I'm not sure why the semantics are different between Glibc and Darwin, but I've added bangs here to resolve the incongruity, not sure if you have another recommendation.

Reply:
Sorry, I mean @weissi above

Reply:
bangs are fine